README.md

```diff
@@ -1 +1,206 @@
-# Recipe-webapp
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+# Recipe webapp
+<b>Recipe webapp</b> is an [SPA](https://en.wikipedia.org/wiki/Single-page_application) that provides a wide library of recipes. The application will allow users to browse though vast recipe categories.
+
+<sup>This project recieves its data from the [EDAMAM.com](https://www.edamam.com/) recipe API. </sup>
+
+</div>
+
+# 📗 Table of Contents
+
+- [Recipe webapp](#-recipe-webapp-)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Recipe webapp ](#-recipe-webapp-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Recipe webapp] <a name="about-project"></a>
+
+**Recipe webapp** is a web application that provides a wide library of recipes. The application will allow users to browse though vast recipe categories based on diets.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Here is a list of the tech used in this project.
+
+  - HTML
+  - CSS
+  - ES6
+  - React
+  - Redux
+  - Jest
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Description of some of the project's key features.
+
+- Home page that displays a list of recipes.
+- Each recipeCard will display an image, a name and a count of ingredients.
+- Each recipeCard will have a button to view the recipe details.
+- Each details page will diplay the recipe's unique information.
+- Has working linters to encourage clean code.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Here is be the link to the demo project as well as a [link to a video](https://www.loom.com/share/16581853ab704d80bc2b4bae09b4fe73) of me explaining more in detail my webapp.
+
+- [Recipe web app](https://recipe-web-app-bbm9.onrender.com) was deployed with the use of render.
```

Comment:
- [ ] [OPTIONAL AND IMPORTANT] I noticed that your live page take so long to load, this is not good UX as it gets boring, try reducing the amount of images pushed to git or try image optimization by converting all jpegs and other image files to `webp` which has smaller size :+1:

Reply:
Thank you for taking the time to review my code. I remember reading on the render documentation that this is because the page gets dropped after 15 minutes of zero traffic to reduce the weight on their servers. In the future I'll try to research for a better hosting platform to deploy my projects in. 👍